pillar/hadoop/CDH.sls

```diff
@@ -15,4 +15,15 @@ log-shipper-patterns:
     - '"/var/log/pnda/impala/*.INFO"'
     - '"/var/log/pnda/impala-llama/*.log"'
   hue:
-    - '"/var/log/pnda/hue/*.log"'
\ No newline at end of file
+    - '"/var/log/pnda/hue/*.log"'
+hadoop_service_name:
```

Comment:
It's unusual to specify things in the Hadoop section but only specify them for one Hadoop version. Please look at how these pillar sections are used elsewhere in the code.

If you need a generic service name abstractor for Hadoop, that's fine - but please use it consistently, I still see e.g. 'oozie01' in the code even though this abstraction has been placed in the pillar.

Reply:
working for the New fix will update once it's completed.